common/js/src/executable-module/executable-module.js

```diff
@@ -54,8 +54,7 @@ ExecutableModule.Toolchain = {
 };
 
 /** @define {string} */
-const TOOLCHAIN = goog.define(
-    'GoogleSmartCard.ExecutableModule.TOOLCHAIN', 'pnacl');
```

Comment:
(To explain, this default here isn't needed anymore, since common.mk now makes sure that the variable is always set to something.)